conops/simulation/reaction_wheel.py

```
@@ -0,0 +1,123 @@
+from __future__ import annotations
+
+import logging
+from math import pi
+from typing import Any
+
+# module logger for optional debug tracing
+logger = logging.getLogger(__name__)
+
+
+def _to_float(val: Any, default: float = 0.0) -> float:
+    """Safely coerce a value to float with a default fallback."""
+    try:
+        return float(val)
+    except Exception:
+        return default
```

Not a fan of silently failing over to zero.